### PR TITLE
Migrate thoth-station test namespace to rick cluster

### PR DESCRIPTION
**Pull request by harshad16**
Migrate thoth-station test namespace to rick cluster
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Fixes: https://github.com/operate-first/support/issues/372

### CODE REVIEW

**Comment by Gregory-Pereira** [comment]
If it's just migrating the namespace then it looks good. I'll let someone else approve to push it through though.

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]
@Gregory-Pereira  in the future, you can also just say '/lgtm' if it looks good to you :) 

/lgtm
/approve

**Pull request merged**
